src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -339,6 +356,8 @@ class SchedulingQueue {
     }
   }
 
+  // Worker context.
```

Comment:
this is owned by CoreWorkerDirectTaskReceiver right?

Reply:
owned by the core worker actually